test/functional/editions_controller_test.rb

```diff
@@ -1125,7 +1125,7 @@ class EditionsControllerTest < ActionController::TestCase
     end
 
     should "update publishing API upon unpublishing an edition" do
-      UnpublishService.expects(:call).with(@guide.artefact, @user, @redirect_url).returns(true)
+      UnpublishService.stubs(:call).with(@guide.artefact, @user, @redirect_url).returns(true)
```

Comment:
What's the reasoning behind this change? It seems from the test name that this `expects` is a key part of the test?